File: ui/src/lib/consts.ts
export enum Outcome {
    Passed = "Passed",
    Failed = "Failed"
}

export enum Analysis {
    NotAnalyzed = "Not Analyzed",
    TruePositive = "True Positive",
    FalsePositive = "False Positive",
    TrueNegative = "True Negative",
    FalseNegative = "False Negative"
}

export enum Resolution {
    Unresolved = "Unresolved",
    TicketCreated = "Ticket Created",
    QuickFix = "Quick Fix",
    KnownIssue = "Known Issue",
    TestFixed = "Test Fixed",
    TestDisabled = "Test Disabled"
}

File: pytest/oar/consts.py
from enum import Enum


class Outcome(str, Enum):
    Passed = "Passed"
    Failed = "Failed"


class Analysis(str, Enum):
    NotAnalyzed = "NotAnalyzed"
    TruePositive = "TruePositive"
    FalsePositive = "FalsePositive"
    TrueNegative = "TrueNegative"
    FalseNegative = "FalseNegative"


class Resolution(str, Enum):
    Unresolved = "Unresolved"
    NotNeeded = "NotNeeded"
    TicketCreated = "TicketCreated"
    QuickFix = "QuickFix"
    KnownIssue = "KnownIssue"
    TestFixed = "TestFixed"
    TestDisabled = "TestDisabled"
